pkg/api/handlers/handler_utils.go

```diff
@@ -49,3 +49,21 @@ func ParseTimeFromStringWithDefault(timeString string, defaultTime time.Time) (t
 	}
 	return parsedTime, nil
 }
+
+func GetProjectSpecStatistics(h *Handler, projectName string) []models.TestSummary {
+	var testSummaries []models.TestSummary
+	h.db.Table("test_runs"). //TODO: move query to handler_utils
```

Comment:
Is the TODO intended for a later issue or part of this one?

Reply:
This was a comment that I'd addressed but forgotten to delete, it is removed now- thanks for the catch